conans/client/build/autotools_environment.py

```diff
@@ -4,39 +4,17 @@
 
 from conans.client import join_arguments
 from conans.tools import environment_append, args_to_string, cpu_count, cross_building, detected_architecture
+from conans.client.build.compiler_flags import architecture_flags, libcxx_flags
 from conans.client.tools.win import unix_path
 from conans.client.tools.oss import OSInfo
 
-sun_cc_libcxx_flags_dict = {"libCstd": "-library=Cstd",
-                            "libstdcxx": "-library=stdcxx4",
-                            "libstlport": "-library=stlport4",
-                            "libstdc++": "-library=stdcpp"}
-
-architecture_dict = {"x86_64": "-m64", "x86": "-m32"}
-
 
 def stdlib_flags(compiler, libcxx):
```

Comment:
I think we could remove the `stdlib_flags` and `stdlib_defines` functions and call directly to the libcxx_flags, right?

Reply:
yes, if it makes sense

Reply:
removed